vpr/src/pack/attraction_groups.cpp

```diff
@@ -0,0 +1,38 @@
+#include "attraction_groups.h"
+
+AttractGroupId AttractionInfo::get_atom_attraction_group(AtomBlockId atom_id) {
```

Comment:
for a short function like this that just returns an int, could make it an inline function in the header file

Reply:
make all the short ones in this file inline in the header file